tests/apps/search/test_indexers_persons.py

```diff
@@ -26,14 +27,30 @@ def test_indexers_persons_get_es_documents(self, _mock_picture):
         Happy path: person data is fetched from the models properly formatted
         """
         person1 = PersonFactory(
+            extended_object__title__language="en",
+            extended_object__title__title="my first person",
             fill_portrait=True,
-            page_title={"en": "my first person", "fr": "ma première personne"},
             should_publish=True,
         )
+        TitleFactory(
+            page=person1.extended_object, language="fr", title="ma première personne"
+        )
+        add_plugin(
```

Comment:
I think it will be easier to understand the code in this case if we don't use `fill_portrait` in the first place.